packages/cli/test/models/compiler/solidity/SolidityProjectCompiler.test.js

```diff
@@ -0,0 +1,39 @@
+require('../../../setup')
```

Comment:
It would be interesting to use the native solc compiler (not solcjs) and compare bytecode output. This could be done easily: https://github.com/ajsantander/solz/blob/master/src/compile/Compiler.js#L33

Reply:
Niceeee, we should! Already added an issue for that (https://github.com/zeppelinos/zos/issues/506)